src/modules/Map/services/mapUtils.js

```
@@ -83,94 +85,120 @@ export const checkContraints = ({
   }, false);
 };
 
-let listenerWaiter = 0;
+export function getInteractionsOnEvent ({
+  eventType,
+  map,
+  point,
+  interactions: eventInteractions,
+}) {
+  let features;
+  try {
+    features = map.queryRenderedFeatures(point);
+  } catch (e) {
+    return null;
```

A comment on why this would fail would be nice

---

I don't know why but sometimes it does.

---

Silent error is not the solution. At least a `console.error()` or `console.info()` should expose issue for dev purpose.

---

This is not the scope of this PR.